spec/support/fixtures/controller_fixtures.cr

```diff
@@ -83,6 +83,7 @@ class HelloController < Amber::Controller::Base
 end
 
 class RenderController < Amber::Controller::Base
+  include Amber::Controller::Helpers::Render
```

Comment:
Should we add an `ApplicationController` fixture in between?

Reply:
Good one, will do, thanks!

Reply:
Actually @drujensen looking into spec/support/fixtures/controller_fixtures.cr I suggest things to remain as they are.

Because if we add ApplicationController fixture then other fixture classes in that file should inherit from it, which would make all of them have more functionality than the spec specifically wanted to test. Which in turn would make it possible that we miss to notice proper encapsulation/separation of code.

So I would for keeping specs as granular and specific as they are now.